carbon-design-system/carbon
# fix(sketch): update icon command with new metadata structure

This PR updates our icon command in our Sketch plugin. It seems like there were some updates between sketch versions that caused breaking changes (namely `Rectangle` and how it's used). In addition, this PR uses our new metadata structure which simplifies some of the logic for generating our icons page.

#### Changelog

**New**

**Changed**

- Update sketch plugin with new metadata structure
- Update sketch plugin wi th changes between sketch versions

**Removed**

#### Testing / Reviewing

- Follow the steps in the Sketch package to link the plugin
- Run the icons sync symbols command
- Run the icons generate page command
- Verify that both run and complete as expected
